### PR TITLE
Load place info in store for place recognition.

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

File: proto/recon.proto [hunk 1]
```diff
@@ -145,3 +145,15 @@ message RecognizePlacesResponse {
...
   repeated Item items = 1;
 }
+
+// RecogPlace contains info of a place for RecognizePlaces method.
+message RecogPlace {
+  repeated string words = 1;
+  string dcid = 2;
+  repeated string containing_places = 3;
+  int64 population = 4;
+}
```
Comment (pradh) [inline proto/recon.proto:155]: Just wonder why this is proto and not struct?  Also, is recon.proto an internal proto

File: proto/recon.proto [hunk 2]
```diff
@@ -145,3 +145,15 @@ message RecognizePlacesResponse {
   }
   repeated Item items = 1;
 }
+
+// RecogPlace contains info of a place for RecognizePlaces method.
+message RecogPlace {
+  repeated string words = 1;
```
Comment (pradh) [inline proto/recon.proto:151]: These parts of the name right?  We may soon want to add multiple names.  So either make it `repeated string names = 1;` that we split,
or 

```
message NameParts {
  repeated string words = 1;
}

repeated NameParts names = 1;
```

**Review by pradh — APPROVED** [review]
Thanks!

File: proto/recon.proto (hunk 1 above)
Comment (Spaceenter) [reply proto/recon.proto:155]: This is to make unit test easier. Deep comparison of proto messages is cleaner than deep comparison of structs.
recon.proto could be public I think? I don't see a reason why we want to keep it private.

File: proto/recon.proto (hunk 2 above)
Comment (Spaceenter) [reply proto/recon.proto:151]: done

**Pull request merged**